var/spack/repos/builtin/packages/occa/package.py

```diff
@@ -50,13 +51,16 @@ class Occa(Package):
             description='Activates support for OpenCL')
     variant('coi',   default=False,
             description='Activates support for COI')
-    depends_on('cuda', when='+cuda')
-    
+    # depends_on('cuda', when='+cuda')
```

Comment:
Thanks @tgamblin, that was exactly what I needed.

@adamjstewart: yes, you have to install a DMG, which on my system requires an administrator password, so the external package certainly makes sense